abtesting/Shared/ContentView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Copyright 2021 Google LLC
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import Firebase
+
+struct ContentView: View {
+  @ObservedObject var appConfig: AppConfig
+  let data: [(title: String, subtitle: String)] = [
+    ("Getting Started with Firebase", "An Introduction to Firebase"),
+    ("Google Firestore", "Powerful Querying and Automatic Scaling"),
```

Comment:
Why these three Firebase products and why isn't ABTesting included?

Reply:
This was just mock data I wrote here as placeholder display stuff that Jesus copy-pasted over.

Reply:
Let's at least add the data for ABTesting since that is the current Quick Start.